Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -323,16 +371,17 @@ private PrivateKey ValidateRawHex(string rawKeyHex)
             return key;
         }
 
-        private Stream PathHandler(string pathString)
+        private string ValidateJsonPath(string jsonPath)
         {
-            if (pathString == string.Empty)
+            try
             {
-                return System.Console.OpenStandardOutput();
+                string json = new StreamReader(jsonPath).ReadToEnd();
+                return json;
             }
-            else
+            catch (Exception)
             {
-                string path = Path.GetFullPath(pathString);
-                return File.Open(path, FileMode.OpenOrCreate);
+                Utils.Error("This is not valid json file or file does not exists.");
```

Comment:
The [`System.IO.StreamReader`][stream-reader] doesn't validate whether the content is compatible with JSON format or not. The original code had used `ProtectedPrivateKey.FromJson` so the error message was right.
So this error message should be corrected like...

```suggestion
                Utils.Error("This file does not exist.");
```

[stream-reader]: https://docs.microsoft.com/ko-kr/dotnet/api/system.io.streamreader?view=net-6.0